Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -250,7 +244,8 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         let tab = tabManager.getTab(indexPath.item)
         let cell = tableView.dequeueReusableCellWithIdentifier(CellIdentifier) as CustomCell
         cell.titleText.text = tab.title
-        cell.background.image = tab.screenshot(size: CGSize(width: tableView.frame.width, height: TabTrayControllerUX.CellHeight))
+        let screenshotAspectRatio = tableView.frame.width / TabTrayControllerUX.CellHeight
+        cell.background.image = screenshotHelper.takeScreenshot(tab, aspectRatio: Float(screenshotAspectRatio), quality: 1)
```

Comment:
Definitely, but that's a different bug! This bug is just about adding about:home support for screenshots.